pkgs/by-name/op/openjpeg/package.nix

```diff
@@ -39,11 +48,13 @@ stdenv.mkDerivation rec {
     "-DBUILD_SHARED_LIBS=ON"
     "-DBUILD_CODEC=ON"
     "-DBUILD_THIRDPARTY=OFF"
-    (mkFlag jpipLibSupport "BUILD_JPIP")
-    (mkFlag jpipServerSupport "BUILD_JPIP_SERVER")
+    (lib.cmakeBool "BUILD_JPIP" jpipLibSupport)
+    (lib.cmakeBool "BUILD_JPIP_SERVER" jpipServerSupport)
     "-DBUILD_VIEWER=OFF"
     "-DBUILD_JAVA=OFF"
-    (mkFlag doCheck "BUILD_TESTING")
+    "-DJPYLYZER_EXECUTABLE=${lib.getExe' jpylyzer "jpylyzer"}"
+    "-DOPJ_DATA_ROOT=../../data"
+    (lib.cmakeBool "BUILD_TESTING" doCheck)
```

Comment:
`BUILD_TESTING` is automatically set, no need to do it ourselves.

Reply:
> `BUILD_TESTING` is automatically set, no need to do it ourselves.

if I don't set it no tests run.

Reply:
Weird. I guess they flip the default or something.